views/reservations/index.haml

```diff
@@ -1,4 +1,4 @@
-="Number of today's reservations: #{@reservations.count}"
+="Number of reservations at #{@day}: #{@reservations.count}"
```

Comment:
`= "..."` should be rewritten as `...`<br>The = symbol should have one space separating it from code